Cargo.lock

```diff
@@ -244,7 +244,7 @@ dependencies = [
 
 [[package]]
 name = "hf_transfer"
-version = "0.1.2"
+version = "0.1.3"
```

Comment:
Did you just change the `Cargo.lock` file? This file shouldn't be touched manually, you should change the `Cargo.toml`

Reply:
No probs, thought you wanted to increment the version and did it in the `.lock` file. No need to revert